### DISCUSSION TITLE
Welcome to Easy-Consulting-react Discussions!

### DISCUSSION THREAD

**Discussion by Ujjalzaman** [Ujjalzaman/Easy-Consulting-react#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ujjalzaman** [comment]
Hey guys, You give me your suggestion for this fullstack project. Or you could contribute

**Reply by wajid-chopdawg** [reply]
hi
